### ISSUE TITLE
Feature request: partial digest generation

### ISSUE THREAD

**Issue by thatlittleboy** [nedbat/dinghy#33]
**Scenario:**  
I have a `dinghy.yaml` file consisting of, say, 5 digests. For testing, I would just like to generate the HTML for one of the digests in the yml file, without having to compose a separate `dinghy-test.yaml` file consisting only of that 1 digest.

I *could* comment out the digests that I don't need in the yaml file, run `dinghy`, then undo the commenting. It's what I've been doing thus far, but maybe a more ergonomic solution is to offer an API like so:

```bash
$ dinghy [OPTIONS] [INPUT yaml] [digest-names]*

# the current API is
# dinghy [OPTIONS] [INPUT yaml]
```

or something to this effect. Or the digest names (1 or more) can be passed in as CLI flags rather than arguments.

Just throwing this idea out here for consideration, it's not an urgent feature request. :) Thanks again for the wonderful tool!

---

To be clear, suppose I have dinghy.yaml

```yaml
digests:
  # ----- Neovim -----
  - digest: digests/nvim-1day.html
    title: Nvim-related news
    since: 1 day
    items: &nvim-items
      - https://github.com/nvim-treesitter/nvim-treesitter/
      - https://github.com/nvim-tree/nvim-tree.lua

  - digest: digests/nvim-3days.html
    title: Nvim-related news
    since: 3 day
    items: *nvim-items

  # ----- Python -----
  - digest: digests/python-1day.html
    title: Python-related news
    since: 1 day
    items: &python-items
      - https://github.com/pola-rs/polars
      - https://github.com/charliermarsh/ruff
      - search: >-
          repo:yt-dlp/yt-dlp
          NOT extractor in:title
          -label:"site-request"
        title: yt-dlp/yt-dlp

  # ----- Mkdocs -----
  - digest: digests/mkdocs-1day.html
    title: Mkdocs-related news
    since: 1 day
    items: &mkdocs-items
      - https://github.com/mkdocstrings/mkdocstrings
      - https://github.com/mkdocstrings/python
      - https://github.com/mkdocstrings/griffe
      - https://github.com/facelessuser/pymdown-extensions
      - https://github.com/squidfunk/mkdocs-material
```

Perhaps I only want to see the `mkdocs` digest, the idea is that I would just run some command that looks like

```
dinghy dinghy.yaml mkdocs-related-news
# or
dinghy --include mkdocs-related-news
```
(I slugify-ed the title, but we could attach a `name` configuration for each digest also, no preference from me)

**Comment by nedbat** [comment]
Good idea! I need this myself!

Implemented in commit 4aae888.

**Comment by nedbat** [comment]
This is now released in dinghy 1.2.0.

**Comment by thatlittleboy** [comment]
That's awesome @nedbat , thank you very much! very helpful :)
